css/globals.css

```diff
@@ -0,0 +1,3 @@
+.newlines {
+  white-space: pre-line;
+}

```

Comment:
What you say makes a lot of sense, but this is a huge change that affects most of the files in the project and I think it would make sense to have this as a separate pull request.

So maybe after the last "Unpack Components" there would be another pull request focusing on component usage, things like this and self-closing tags and getting rid of the custom CSS.

I almost feel like some of the past commits (e.g. replacing `<b>` with `<strong>`) would make more sense in that pull request too, might be good to move them there.
What do you think?

Reply:
+1, can put that in the other PR.